src/server/api/routers/invite.ts

```
@@ -0,0 +1,283 @@
+
+        // Prevent self-invite
+        if (inviteCode.creatorId === userId) {
+          throw new TRPCError({
+            code: "BAD_REQUEST",
+            message: "You cannot use your own invite code.",
+          });
+        }
+
+        await tx.inviteCode.update({
+          where: { id: inviteCode.id },
+          data: { currentRedemptions: { increment: 1 } },
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'data' should be before 'where'.

(sort-keys-fix/sort-keys-fix)